koku/api/report/ocp/query_handler.py

```diff
@@ -152,6 +165,7 @@ def _format_query_response(self):
 
         """
         output = self._initialize_response_output(self.parameters)
+        output = self._check_if_distributed_overhead(output)
```

Comment:
Rather than mutate the passed in data, a clearer interface would be to return a boolean and have the caller set the value.
```suggestion
        output = self._check_if_distributed_overhead(output)
        output["distributed_overhead"] = self._check_if_distributed_overhead(output)
```

Reply:
In this case though, we only set `output["distributed_overhead"]` when `self._report_type == "costs_by_project"`... Though, I suppose that bit can be moved here instead of inside of _check_if_distributed_overhead.

Reply:
I went ahead and moved the logic here instead.